src/ucp/core/ucp_am.h

```diff
@@ -11,11 +11,6 @@
 
 typedef union {
     struct {
-        uint32_t     length;      /* length of an AM. Ideally it would be size_t
-                                   * but we want to keep this struct at 64 bits
-                                   * to fit in uct_ep_am_short header. MAX_SHORT
-                                   * or b/zcopy MTU
-                                   * should be much smaller than this anyway */
```

Comment:
maybe add uint32_t padding ?so it could be set to 0 in pack header function

Reply:
ok